build_tools/patch_deps_unix/hppfcl.patch

```diff
@@ -4,14 +4,14 @@ index d675c2c..c503cf1 100644
 +++ b/CMakeLists.txt
```

Comment:
Modifications included my mistake.

Reply:
Not really - these are trailing whitespaces automatically removed, I thought it would be worth including.

Reply:
No, as they are present in the original file. This is a git patch to apply on SOUP. They are the one having those trailing whitespaces and removing them here is just messing up with git. It is working in practice because I'm applying the patches using the command `git apply --reject --whitespace=fix`, but it is not very robust, especially on windows.